pages/tools/generator.js

```diff
@@ -1,11 +1,12 @@
-import GenericLayout from '../components/layout/GenericLayout'
-import Button from '../components/buttons/Button'
-import IconDocuments from '../components/icons/Documents'
-import IconCode from '../components/icons/Code'
-import IconPowerPlug from '../components/icons/PowerPlug'
-import IconRocket from '../components/icons/Rocket'
-import GithubButton from '../components/buttons/GithubButton'
-import GeneratorInstallation from '../components/GeneratorInstallation'
+import GenericLayout from '../../components/layout/GenericLayout'
+import IconDocuments from '../../components/icons/Documents'
+import IconCode from '../../components/icons/Code'
+import IconPowerPlug from '../../components/icons/PowerPlug'
+import GithubButton from '../../components/buttons/GithubButton'
+import GeneratorInstallation from '../../components/GeneratorInstallation'
+
+//import Button from '../../components/buttons/Button'
```

Comment:
why did you decide to comment them out instead of removing?

Reply:
I thought may be for future reference in case reverting back to Button from GithubButton we may need it. 
I will remove it.